samples/excel/89-preview-apis/document-properties.yaml

```diff
@@ -0,0 +1,118 @@
+order: 6
+id: excel-document-properties
+name: Document properties
+description: Get and set document properties.
+host: EXCEL
+api_set:
+    ExcelApi: 1.7
+script:
+    content: |+
+        $("#get-doc-properties").click(() => tryCatch(getDocProperties));
+        $("#set-get-doc-properties").click(() => tryCatch(setGetDocProperties));
+
+        async function getDocProperties() {
+            await Excel.run(async (context) => {
+                let docProperties = context.workbook.properties;
+
+                // Load the read-only document properties.
+                docProperties.load("author, lastauthor, revisionnumber, creationdate");
+
+                await context.sync();
+
+                console.log("Author  :" + docProperties.author);
+                console.log("Last author  : " + docProperties.lastAuthor);
+                console.log("Revision number : " + docProperties.revisionNumber);
+              
+                OfficeHelpers.UI.notify("Workbook creation date : " + docProperties.creationDate.toDateString());            
+            });
+        }
+
+        async function setGetDocProperties() {
+            await Excel.run(async (context) => {
+                let titleValue = "Excel document properties API";
+                let subjectValue = "Set and get document properties";
+                let keywordsValue = "Set and get operations";
+                let commentsValue = "This is an Excel document properties API code sample";
+                let categoryValue = "Office Add-ins";
+                let managerValue = "John";
+                let companyValue = "Microsoft";
+
+                let docProperties = context.workbook.properties;
+
+                // Set the docoment properties.
+                docProperties.title = titleValue;
+                docProperties.subject = subjectValue;
+                docProperties.keywords = keywordsValue;
+                docProperties.comments = commentsValue;
+                docProperties.category = categoryValue;
+                docProperties.manager = managerValue;
+                docProperties.company = companyValue;
+
+                await context.sync();
+                
+                // Get the docoment properties.
+                // To learn how to view document properties in the Ui, 
+                // see https://support.office.com/en-us/article/View-or-change-the-properties-for-an-Office-file-21D604C2-481E-4379-8E54-1DD4622C6B75
+                console.log("Title :" + docProperties.title);  
```

Comment:
The code here is just reading back the local variable. It's not reading the actual `context.workbook.properties`.

I think what you need is a method that sets `context.workbook.properties`. Then have an instruction in the taskpane that tells the user to click the **Get document properties** button a second time.